server/src/main/webapp/WEB-INF/rails/webpack/views/pages/pipeline_activity/spec/comment_widget_spec.tsx

```diff
@@ -207,7 +208,7 @@ describe("CommentWidget", () => {
       helper.clickByTestId("edit-comment-button", modal);
 
       expect(helper.byTestId("textarea-to-add-or-edit-comment", modal)).toBeInDOM();
-      expect((helper.byTestId("textarea-to-add-or-edit-comment", modal) as HTMLInputElement).value).toBe(comment());
+      expect($("textarea").val()).toBe(comment());
```

Comment:
Jquery??. I thought we are going away from it.

Reply:
Fixed!!